.php_cs.dist.php

```diff
@@ -12,7 +12,7 @@
 
 return (new PhpCsFixer\Config())
     ->setRules([
-        '@PSR2' => true,
+        '@PSR12' => true,
```

Comment:
Nice!

Reply:
I'll submit a PR to laravel skeleton too